### PR TITLE
#781 Document Front interface in README.md

### PR DESCRIPTION
**Pull request by t-izbassar**
Implementation for #781.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#803](https://github.com/yegor256/takes/issues/803) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#803](https://github.com/yegor256/takes/issues/803) is assigned to @g4s8[/z](https://www.0crat.com/u/g4s8), here is [why](https://www.0crat.com/footprint/C3FFK3YAY/472122845). The budget is 15 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§27](http://datum.zerocracy.com/pages/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

File: README.md [hunk 1]
```diff
@@ -314,6 +315,27 @@ public final class App {
 }
 ```
 
+## Front interface
+
+Essential part of [Bigger Example](#a-bigger-example) is 
+[Front](src/main/java/org/takes/http/Front.java) interface.
+It is representing an HTTP front that will start the server,
```
Comment (g4s8) [inline README.md]: @t-izbassar actually this `Front` can start non-http servers too ;) I'd just say "front" - is an object which encapsulates server's "back" and can be started.

File: README.md [hunk 2]
```diff
@@ -314,6 +315,27 @@ public final class App {
...
+which will accept HTTP requests and return results. `FtBasic`,
+which is a basic front, implements that interface - you've
+seen it's usage in above mentioned example.
+
+There are other useful implementations of this interface: 
+
+* The [FtRemote](src/main/java/org/takes/http/FtRemote.java)
+class allow you to provide script, that will be executed against
```
Comment (g4s8) [inline README.md]: ~~allow~~ allows

**Comment by t-izbassar** [comment]
@g4s8 thanks for input. Fixed.

**Comment by t-izbassar** [comment]
@yegor256 ping

**Comment by t-izbassar** [comment]
@yegor256 ping

**Comment by t-izbassar** [comment]
@yegor256 can you merge this one?

**Comment by yegor256** [comment]
@rultor merge

**Comment by t-izbassar** [comment]
@yegor256 I feel like the external XML was changed. So every build will fail. Reported here: #805 We need to ignore that particular test in order to proceed.

**Comment by t-izbassar** [comment]
@yegor256 can you please commit hot-fix in master, ignoring the `RsXsltTest.loadsExternalImports()` test case and re-trigger the merges? The #805 about failure is added I can also specifically mention, that `@Ignore` should be deleted on that test case. 

This is obvious that failure is not related to the changes, as only comment and README.md were modified.

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
@elenavolokhova[/z](https://www.0crat.com/u/elenavolokhova) please review this job, as in [§30](http://datum.zerocracy.com/pages/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by elenavolokhova** [comment]
@g4s8 According to our [QA rules](http://datum.zerocracy.com/pages/policy.html#42)
> The code reviewer found at least three problems in the code.

Only two issues were found during review.
Please, **confirm** that you'll try to find at least three design problems in next reviews.

**Comment by g4s8** [comment]
@elenavolokhova sure, I'll try to find it

**Comment by elenavolokhova** [comment]
@g4s8 Thank you!

**Comment by elenavolokhova** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
QA review completed: +15 points just awarded to @elenavolokhova[/z](https://www.0crat.com/u/elenavolokhova), total is [+45](https://www.0crat.com/u/elenavolokhova)

**Comment by 0crat** [comment]
Order was finished, quality was "acceptable": +15 points just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8), total is [+4480](https://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://datum.zerocracy.com/pages/policy.html#28): +15 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256), total is [+10504](https://www.0crat.com/u/yegor256)

**Pull request merged**